### DISCUSSION TITLE
Image tracking support

### DISCUSSION THREAD

**Discussion by devhims** [google/model-viewer#2090] [Ideas]
Can we expect image tracking support to be extended to model viewer (via Scene Viewer) in the near future?

**Comment by elalish** [comment]
Doubtful? It hasn't made it into the WebXR API yet. @tpsiaki can comment on SceneViewer plans.

**Reply by devhims** [reply]
It'd be really helpful if model viewer extends some of the AR.js APIs as well. It'll be an important step in making it a complete solution for AR on the web.  :)

**Comment by fabiotgolo** [comment]
This is really important for marketing. We could apply AR in labels and banners.
